code/modules/cm_marines/Donator_Items.dm

```diff
@@ -1319,10 +1320,10 @@
 	item_state = "doom_boots"
 
 /obj/item/clothing/shoes/marine/fluff/steelpoint //CKEY=steelpoint (UNIQUE)
-	name = "M4X Boot"
-	desc = "Boots to the M4X.  DONOR ITEM"
-	icon_state = "jackboots"
-	item_state = "jackboots"
+	name = "M4-X Boot"
+	desc = "Standard issue boots issued alongside M4-X armour, features a special coating of acid-resistant layering to allow its operator to move through acid-dretched enviroments safely. This prototype version lacks that feature.  DONOR ITEM"
```

Comment:
Inconsistency - armour and armor are used interchangeably.

Reply:
Curses of my Australian language strike again